packages/replayio/src/commands/upload-source-maps.ts

```diff
@@ -1,22 +1,21 @@
+import { uploadSourceMaps as uploadSourceMapsExternal } from "@replayio/sourcemap-upload";
+import { replayApiServer } from "../config";
+import { getAccessToken } from "../utils/authentication/getAccessToken";
 import { registerCommand } from "../utils/commander/registerCommand";
 import { exitProcess } from "../utils/exitProcess";
 import { dim } from "../utils/theme";
+import { logPromise } from "../utils/async/logPromise";
 
 registerCommand("upload-source-maps <paths...>", { requireAuthentication: true })
+  .alias("upload-sourcemaps")
```

Comment:
I think we should remove this.

Reply:
I don't think it adds much value and I think it slightly worsens the first impression of the library:
![Screenshot 2024-04-12 at 10 04 16 AM](https://github.com/replayio/replay-cli/assets/29597/50d70e68-7914-40bc-bd1f-5bfc4b827802)

Reply:
It might help people to migrate to the new CLI - since the command name was changed. What do you think about dropping aliases from this table? I didn't realize that it would be displayed here.

Note that without this we get a good suggestion from `commander`:
```
error: unknown command 'upload-sourcemaps'
(Did you mean upload-source-maps?)
```
That said, I would consider this to be a likely error so I thought it would be nice to handle this gracefully and deal with it. Up to you though - I can make any requested changes.